_pages/attributes.md

```diff
@@ -6,6 +6,11 @@ lead: >
 
 Here are the possible attributes that can be requested at a given IAL. This table contains the available user attributes, the IAL they are associated with, and how they can be accessed in OpenID Connect and SAML.
 
+It is important to expect any number of characters in the `(string)` datatype unless directly followed by a number such as `(string36)`.
+
+The following fields are stored together in a text datatype with a maximum length of 65,535 bytes:<br>
+First name, Last name, Address, Phone, Date of birth, Social security number
```

Comment:
Yes, we will be re-working the dev docs site soon following the structure in the brochure and partners site.

What about saying this instead? I'm trying to be as transparent as possible.
```suggestion
It is important to expect any number of characters in the `(string)` datatype unless directly followed by a number such as `(string36)`. Strings are encrypted and stored in a text datatype with a maximum length of 65,535 bytes.
```